secret_store/src/key_server_set.rs

```diff
@@ -168,6 +168,77 @@ impl ChainNotify for OnChainKeyServerSet {
 	}
 }
 
+trait KeyServerSubset<F: Fn(Vec<u8>) -> Result<Vec<u8>, String>> {
```

Comment:
@svyatonik please review changes in secret store. I made them, cause it was the easiest way to replace old contracts interface with the new one

Reply:
Thanks! Code looks perfect! Right now I'm updating SS service contract && will do real-world testing of these changes (all three contracts) before making a PR.